property/index.js

```diff
@@ -78,6 +80,7 @@ module.exports = yeoman.generators.Base.extend({
         name: 'type',
         message: 'Property type:',
         type: 'list',
+        default: this.propDefinition && this.propDefinition.type,
```

Comment:
Well, we keep beating around the same bush: the inadequacy of using that item typed array syntax that's becoming a total nuisance to handle... even despite the fact that even inquirer handles the array structure in a type, itemType tuple...